src/config.js

```diff
@@ -390,6 +391,12 @@ function preprocessConfig(cfg) {
     });
     cfg["channel-blacklist"] = tbl;
 
+    /* Check channel path */
+    if(!/^[-\w]+$/.test(cfg["channel-blacklist"])){
+        LOGGER.warn("Channel paths may only use the same characters as usernames and channel names.");
```

Comment:
`LOGGER.error` and explicitly call `process.exit(1)` to prevent it from just being caught by the uncaught exception handler and chugging along.